GSA/made-in-america
# add faq with links

Add new FAQ to the faq list.

How to test
Pull this specific branch and start the project locally.
Navigate to (http://127.0.0.1:4000/) and check if the additional FAQ is there and check the content and links associated with it.

Ticket: https://cm-jira.usa.gov/browse/MIA-497
